pkg/apis/core/validation/validation.go

```diff
@@ -3773,35 +3775,54 @@ func podSpecHasContainer(spec *core.PodSpec, containerName string) bool {
 }
 
 const (
+	// a sysctl name regex
+	SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+
+	// a sysctl name regex contains a slash
+	SysctlRelaxedFmt string = "(" + SysctlSegmentContainSlashFmt + "\\.)*" + SysctlSegmentContainSlashFmt
+
 	// a sysctl segment regex, concatenated with dots to form a sysctl name
 	SysctlSegmentFmt string = "[a-z0-9]([-_a-z0-9]*[a-z0-9])?"
 
-	// a sysctl name regex
-	SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+	// a sysctl segment regex contains a slash, concatenated with dots to form a sysctl name
+	SysctlSegmentContainSlashFmt string = "[a-z0-9]([-_/a-z0-9]*[a-z0-9])?"
 
 	// the maximal length of a sysctl name
 	SysctlMaxLength int = 253
 )
 
 var sysctlRegexp = regexp.MustCompile("^" + SysctlFmt + "$")
 
+var sysctlRelaxedRegexp = regexp.MustCompile("^" + SysctlRelaxedFmt + "$")
+
 // IsValidSysctlName checks that the given string is a valid sysctl name,
 // i.e. matches SysctlFmt.
-func IsValidSysctlName(name string) bool {
+func IsValidSysctlName(name string, isContainSlash bool) bool {
```

Comment:
keep the parameter and the package variable names consistent (use "relaxed" for both, or use "canContainSlash" for both)

Reply:
Updated.